playwright/src/test/java/com/microsoft/playwright/TestPageInterception.java

```diff
@@ -139,15 +139,4 @@ void shouldNotFollowRedirectsWhenMaxRedirectsIsSetTo0InRouteFetch() {
     page.navigate(server.PREFIX + "/foo");
     assertTrue(page.content().contains("hello"));
   }
-
-  @Test
-  void shouldProperlyHandleCharacterSetsInGlobs() {
```

Comment:
Let's port 'should work with glob' from upstream, to make sure our logic doesn't break when constructing java regex from the node's string.